### PR TITLE
Add support for 'base36' and 'base36upper' multibase encodings

### PR DESCRIPTION
**Pull request by cardoso-neto**
I was perusing [multiformats/py-multibase's fork network](https://github.com/multiformats/py-multibase/network) and found this great work by [pinnaculum](https://github.com/pinnaculum/py-multibase/tree/base36).

What do you guys think?

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.0.1
+current_version = 1.0.2
```
Comment (dhruvbaldawa) [inline setup.cfg:2]: I would not update these, let `bumpversion` handle them automatically.

File: setup.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 setup(
     name='py-multibase',
-    version='1.0.1',
+    version='1.0.2',
```
Comment (dhruvbaldawa) [inline setup.py:32]: SAA

File: multibase/__init__.py [hunk 3]
```diff
@@ -4,6 +4,6 @@
 
 __author__ = """Dhruv Baldawa"""
 __email__ = 'dhruv@dhruvb.com'
-__version__ = '1.0.1'
+__version__ = '1.0.2'
```
Comment (dhruvbaldawa) [inline multibase/__init__.py:7]: SAA

File: HISTORY.rst [hunk 4]
```diff
@@ -1,6 +1,11 @@
 History
 -------
 
+1.0.2 (2020-10-08)
```
Comment (dhruvbaldawa) [inline HISTORY.rst:4]: I would prefer updating this to when we release the version

**Review by dhruvbaldawa — APPROVED** [review]
Apart from the version number changes, everything else looks good :+1:

Approving, but please make those version number changes.

**Comment by pinnaculum** [comment]
Hello,

Sorry i was just about to create a pull request from the base36 branch when i saw that @cardoso-neto had already done it :+1:

Added base36 because now go-ipfs (since version 0.7) uses base36 for IPNS keys by default and since [galacteek](https://github.com/pinnaculum/galacteek) uses go-ipfs v0.7 now i needed base36 support so that py-cid can load base36-encoded CIDv1 (it works).

Just removed the version bumps (i needed to change at least the version in setup.py because i use my fork as an external dependency).

Thanks for py-multibase and py-cid :+1 Tell me if something else is needed.

**Comment by dhruvbaldawa** [comment]
Thank you for your contribution! I have approved the PR, you can merge the PR and I can then release it, or if you wish I can do that as well :)

**Comment by pinnaculum** [comment]
> Thank you for your contribution! I have approved the PR, you can merge the PR and I can then release it, or if you wish I can do that as well :)

You're welcome. Since @cardoso-neto created the PR i can't merge it, you should probably do it. Will be nice to have 1.0.2 with base36 on pypi.

**Pull request merged**